src/tests/Dashboard/action/dictionaryAction.test.js

```diff
@@ -111,6 +111,24 @@ describe('Test suite for dictionary actions', () => {
       expect(store.getActions()).toEqual(expectedActions);
     });
   });
+
+  it('should handle fetch dictionaries network error', () => {
+    moxios.wait(() => {
+      const request = moxios.requests.mostRecent();
+      request.reject({
+        status: 599,
```

Comment:
Why status `599`?

Reply:
> Why status `599`?

599 status code was used to create a network timeout so that I can test the behavior when there is no internet.